plugins/common/zpool.sh

```diff
@@ -122,3 +122,8 @@ for zp in $pools; do \
             }
         }'
 done
+
+# pool fragmentation
+zpool list -H -o name,fragmentation | sed 's/\%//g' | while read fraglist; do
+    echo $fraglist | awk {'printf("%s`fragmentation\tL\t%s\n", $1, $2)'};
```

Comment:
The munging of zpool output can be done in a single awk invocation. We can use a substring assignment instead of sed to drop the last character of column 2:

/sbin/zpool list -H -o name,fragmentation | \
  awk '{ frag=substr($2, 1, length($2)-1); printf("%s`fragmentation\tL\t%d\n", $1, frag); }'

I'd also like to see tests on ZoL and FreeBSD just to make sure we haven't accidentally baked in any illumos-specific assumptions.